### PR TITLE
BaSP-TG-103: update admin form 

### PR DESCRIPTION
**Pull request by laura-brussa12**
- Add library and setup inputs
- Create validations following BE with JOI 
- Capitalize values before sending data to BE 
- Update relative routes 
- Test features

### CODE REVIEW

File: src/components/Admin/index.js [hunk 1]
```diff
@@ -1,41 +1,65 @@
...
+    .min(8)
+    .max(12)
+    .pattern(/[a-zA-Z]/)
+    .pattern(/[0-9]/)
+    .required()
+});
+
 const AdminForm = () => {
+  const { handleSubmit, control, setValue } = useForm({
+    mode: 'onChange',
```
Comment (GuidoCerioni) [inline src/components/Admin/index.js:32]: you should use default mode (onSubmit)

**Review by GuidoCerioni — CHANGES_REQUESTED** [review]
See comment!

**Review by Facund0Fernandez — APPROVED** [review]

**Review by FiolAxel — DISMISSED** [review]

**Review by GuidoCerioni — DISMISSED** [review]
Nice job Lau!

**Review by HernanCazeneuve — APPROVED** [review]

**Review by GuidoCerioni — APPROVED** [review]
Nice work!

**Review by GuidoCerioni — APPROVED** [review]

**Review by HernanCazeneuve — APPROVED** [review]

**Review by Gearyandres94 — APPROVED** [review]

**Comment by GuidoCerioni** [comment]
Hi Lau, i merged your PR after it passes testing, but with a change in utils/formatters. It solves a conflict in all others PRs caused by my merged PR. Hope you don't mind.

**Pull request merged**